analyzer/analyzer.spicy

```diff
@@ -61,7 +61,7 @@ type HardwareType = enum {
 
 # RFC-1497: BOOTP Vendor Information Format.
 type Options = unit {
-    : b"\x63\x82\x53\x63"; # Magic Cookie.
+    : skip b"\x63\x82\x53\x63"; # Magic Cookie.
```

Comment:
I thought I might have commented elsewhere, is there a feature request to imply `skip` for anonymous fields without hooks?

Reply:
I don't think so. Deciding whether a field does not have hooks requires global analysis (hooks might be in other files) at the point where we generate the field parsing; this is not something we currently do.

Reply:
https://github.com/zeek/spicy/issues/1872